cmd/modulectl/create/module/flags.go

```diff
@@ -0,0 +1,60 @@
+package module
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/kyma-project/modulectl/internal/service/module"
+)
+
+const (
+	// TODO: Can be extracted to one higher level
+	ModuleConfigFileFlagName    = "module-config-file"
+	ModuleConfigFileFlagDefault = "module-config.yaml"
+	moduleConfigFileFlagUsage   = "Specifies the module configuration file"
+	// ====================================
+
+	CredentialsFlagName    = "credentials"
+	credentialsFlagShort   = "c"
+	CredentialsFlagDefault = ""
+	credentialsFlagUsage   = "Basic authentication credentials for the given repository in the <user:password> format"
+
+	GitRemoteFlagName    = "git-remote"
+	GitRemoteFlagDefault = "origin"
+	gitRemoteFlagUsage   = `Specifies the remote name of the wanted GitHub repository. For Example "origin" or "upstream" (default "origin")`
+
+	InsecureFlagName    = "insecure"
+	InsecureFlagDefault = false
+	insecureFlagUsage   = "Uses an insecure connection to access the registry"
+
+	TemplateOutputFlagName    = "output"
+	templateOutputFlagShort   = "o"
+	TemplateOutputFlagDefault = "template.yaml"
+	templateOutputFlagUsage   = `File to write the module template if the module is uploaded to a registry. (default "template.yaml")`
+
+	RegistryURLFlagName    = "registry"
+	RegistryURLFlagDefault = ""
+	registryURLFlagUsage   = "Context URL of the repository. The repository URL will be automatically added to the repository contexts in the module descriptor."
+
+	//nolint:gosec // Not hardcoded credentials, rather just flag name
+	RegistryCredSelectorFlagName    = "registry-cred-selector"
+	RegistryCredSelectorFlagDefault = ""
+	//nolint:gosec // Not hardcoded credentials, rather just flag name
+	registryCredSelectorFlagUsage = `Label selector to identify an externally created Secret of type "kubernetes.io/dockerconfigjson". It allows the image to be accessed in private image registries. It can be used when you push your module to a registry with authenticated access. For example, "label1=value1,label2=value2".`
+
+	// TODO: Extract one level above
```

Comment:
same as [here](https://github.com/kyma-project/modulectl/pull/40/files#r1734547023). Generally recommendation: if you have some code refactoring concerns that can't resolved in this PR, write a concrete issue instead of `TODO`

Reply:
Also, while we are at it: Please check the repo for leftover `TODO`s, convert them into issues and then enable the TODO linter in the `.golangci.yaml`, I think its the `godox` one.

Reply:
Please also mention created issues in this PR where the TODO has been removed.